.github/workflows/test-security-hub-scan.yaml

```diff
@@ -0,0 +1,47 @@
+name: Test security-hub-scan
+
+on:
+  workflow_call: ~
+
+
+jobs:
+  run-test:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      id-token: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+      # run setup to get uds to use zarf
+      - name: Environment setup
+        uses: ./.github/actions/setup
+
+      - name: build example package
+        id: setup
+        run: |
+          uds run create-nginx-package --no-progress
+          echo "path="$(ls *.zst) >> "$GITHUB_OUTPUT"
```

Comment:
we should avoid wildcarding - this will not pickup application repos that have multiple packages properly (i.e. mattermost which has the main app and a plugins package)